src/common/components/cards/rich-resolution-card-row.tsx

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { CardRowProps } from 'common/configs/unified-result-property-configurations';
+import { NamedFC } from 'common/react/named-fc';
+import { UnifiedRichResolution } from 'common/types/store-data/unified-data-interface';
+import * as React from 'react';
+import { RichResolutionContent } from './rich-resolution-content';
+import { SimpleCardRow } from './simple-card-row';
+
+export interface RichResolutionCardRowProps extends CardRowProps {
+    propertyData: UnifiedRichResolution;
+}
+
+export const RichResolutionCardRow = NamedFC<CardRowProps>(
```

Comment:
Should this specify `RichResolutionCardRowProps`?

```suggestion
export const RichResolutionCardRow = NamedFC<RichResolutionCardRowProps>(
```